crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -201,6 +201,22 @@ def compute_resources_available?
         true
       end
 
+      # Check for presence of HA setup, which is a requirement for non-disruptive upgrade
+      def ha_presence_check
```

Comment:
we don't have any crowbar instance anymore, just class methods. so the errors hash is not an option here